### DISCUSSION TITLE
Etsy API V3 Receipt Progress Step

### DISCUSSION THREAD

**Discussion by justinrowan** [etsy/open-api#1008] [Q&A]
I'm working on a small app for my shop to use internally, and part of that involves querying open receipts. In our shop we've added a couple of unique "progress steps" aside from "New" and "Completed", and I can't find anything in the documentation that lets me query those progress steps (or change them). 

Am I missing something, or is this in the pipeline?

**Comment by Gareth064** [comment]
Hey @justinrowan 

Those custom statuses are available via the API.

I and others brought this up nearly 2 years ago and were told it was on their internal discussion list.

Never heard anything since.

Might be worth email developer@etsy.com to ask about it again. If you get any sort of response then please let the rest of know by updating this discussion

**Reply by justinrowan** [reply]
Thanks @Gareth064, I'm guessing that was a typo and you meant they aren't available in the API? I'll shoot an email to developer@etsy.com with the same question.

**Reply by Gareth064** [reply]
Yup twas a typo 🤣

**Reply by flmu** [reply]
Any news?

**Reply by flmu** [reply]
Reply from Etsy: `At this time no endpoints exist to customize statuses via API. We currently do not have any plans to add this access to our API. But, I have noted your request. We rely on feedback from our members as we continue to make improvements in the best interest of the Etsy Community. `

I think it would help if more people write an email to [developer@etsy.com](mailto:developer@etsy.com) and asked about this feature

**Reply by ritsybits** [reply]
Ha! I just sent them an email about this *before* I saw this group!

**Reply by jbrown17** [reply]
It's been over a year since this was originally posted. Does anyone know of any movement on this? It's pretty disappointing that this isn't implemented. It seems like a pretty glaring hole.

**Reply by Gareth064** [reply]
Don't hold your breath waiting for this @jbrown17. I highly doubt Etsy will ever implement this. There is close to zero engagement from their devs, or a road map, or anything to suggest they are working on any community requests. There are other requests easier than this which are close to 3 years old which have never been done.

**Reply by justinrowan** [reply]
Bumping this for maybe some additional visibility 2 years later. This seems like such an obvious thing to add for those of us who use custom progress steps to track the stages of production/fulfillment. The backend supports it so it should be a relatively minor change to the API to make this information available.

**Reply by jbrown17** [reply]
@justinrowan yep, still would love to have this.
